packages/storybook-components/src/WithDocsCustom.js

```diff
@@ -16,12 +17,10 @@ const PreviewComponent = ({ children }) => (
   </div>
 );
 
-const WithDocsCustom = withDocs({
-  PreviewComponent,
-});
+const withDocsCustom = (readme, ...rest) => withDocs({ PreviewComponent })(navigator.userAgent.match(/Chromatic/) ? '' : readme, ...rest);
 
 PreviewComponent.propTypes = {
   children: PropTypes.node.isRequired,
 };
 
-export default WithDocsCustom;
+export default withDocsCustom;
```

Comment:
Please could you or @Loque- create a ticket to address HOCs being named appropriately and not filed under 'Components'
- [ ] ticket created